Scripts/script-QRadarClassifier.yml

```diff
@@ -104,5 +104,6 @@ dependson:
 outputs:
 - contextPath: Classifier/Description
   description: The QRadar offense 'High level category'
-  important: true
-  importantDescription: The QRadar offense 'High level category'
+important:
+- contextPath: Classifier/Description
+  description: The QRadar offense 'High level category'
```

Comment:
Shorten description
Make sure that value in context is a descriptive sentence why the classifier chose to do that